src/streaming/controllers/PlaybackController.js

```diff
@@ -268,20 +268,22 @@ MediaPlayer.dependencies.PlaybackController = function () {
             if (e.data.index === startIdx) {
                 firstAppended[id] = firstAppended[id] || {};
                 firstAppended[id][type] = true;
-                firstAppended.ready = !((stream.hasMedia("audio") && !firstAppended[id].audio) || (stream.hasMedia("video") && !firstAppended[id].video));
+                firstAppended[id].ready = !((stream.hasMedia("audio") && !firstAppended[id].audio) || (stream.hasMedia("video") && !firstAppended[id].video));
```

Comment:
Please correct me if I am wrong, but won't changing `firstAppended.ready` to be per `id` cause the seek to happen twice?